pypit/arparse.py

```diff
@@ -1690,6 +1701,31 @@ def trace_dispersion_direction(self, v):
                        "0 or 1 (if the dispersion axis is along a row or column respectively)")
         self.update(v)
 
+    def trace_object_function(self, v):
+        """ What function should be used to trace the object in each order?
+
+        Parameters
+        ----------
+        v : str
+          value of the keyword argument given by the name of this function
+        """
+        allowed = ['polynomial', 'legendre', 'chebyshev']
```

Comment:
There are quite a few instances of this throughout arparse.py...

I had originally intended to write a script that would automatically generate a .rst file, containing the doc string for each argument combined with the allowed values. So, if a new parameter is added, we wouldn't have to also update the docstring. I don't think the typical user will go to the docstring to find the allowed values, but is more more likely to go to the documentation. Do you have a preference here?